### PR TITLE
Solver keyed fdb export

### PR DESCRIPTION
**Pull request by cderb**
Export fdb with the top 4 solvers.

### CODE REVIEW

File: tuna/miopen/subcmd/export_db.py [hunk 1]
```diff
@@ -281,9 +316,9 @@ def export_fdb(dbt, args):
   """Function to export find_db to txt file
   """
   query = get_fdb_query(dbt, args)
-  fdb_alg_lists = get_fdb_alg_lists(query)
-  miopen_fdb = build_miopen_fdb(fdb_alg_lists)
```
Comment (alexandraBara) [inline tuna/miopen/subcmd/export_db.py:285]: If this is not called anywhere anymore, please remove

File: tuna/miopen/subcmd/export_db.py [hunk 2]
```diff
@@ -255,6 +255,40 @@ def build_miopen_fdb(fdb_alg_lists):
   return miopen_fdb
 
 
+def build_miopen_fdb_2(query):
```
Comment (alexandraBara) [inline tuna/miopen/subcmd/export_db.py]: this will need a utest, how does this pass the codecov?

**Review by alexandraBara — CHANGES_REQUESTED** [review]

File: tuna/miopen/subcmd/export_db.py (hunk 2 above)
Comment (cderb) [reply tuna/miopen/subcmd/export_db.py]: There was no previous unit test, so the change did not register.

**Comment by cderb** [comment]
If #894 is merged first I will handle the resolution here.

**Review by alexandraBara — APPROVED** [review]
I see where you and @reidkwja  are stepping on each others toes. @reidkwja  please integrate this branch/utest in your export_db branch. Make sure to keep your own yaml testing however.

File: tests/test_export_db.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+from tuna.utils.db_utility import DB_Type
+from utils import add_test_session, DummyArgs
+
+sys.path.append("../tuna")
+sys.path.append("tuna")
+
+this_path = os.path.dirname(__file__)
+
+
+def test_export_db():
```
Comment (reidkwja) [inline tests/test_export_db.py:43]: I believe we can merge both testing files into one without removing much code...My tests are more focused on individual functional runs and callability while your test_export_db is more integrative and focused on ensuring that all functions run correctly relative to each others with the correct data parsing.

**Review by reidkwja — APPROVED** [review]

**Pull request merged**